src/node_crypto.cc

```diff
@@ -751,6 +751,23 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
   CHECK_EQ(sc->ca_store_, nullptr);
 
   if (!root_cert_store) {
+#if defined(NODE_SYSTEM_CERT_STORE)
+    // *Assume* OpenSSL is setup correctly, which is the case
+    // for distribution supplied versions.
```

Comment:
That's only going to be the case when node is linked against the distro openssl.  It defaults to /usr/local/ssl with the bundled copy and that's unlikely to be the right path.
 
Aside: s/setup/set up/